moped-database/migrations/1727458564792_add_funding_program_to_project_list_view/down.sql

```diff
@@ -0,0 +1,545 @@
+-- Could not auto-generate a down migration.
+-- Please write an appropriate down migration for the SQL below:
+DROP VIEW IF EXISTS exploded_component_arcgis_online_view;
+DROP VIEW IF EXISTS component_arcgis_online_view;
+DROP VIEW IF EXISTS project_list_view;
+--
+CREATE OR REPLACE VIEW project_list_view AS WITH project_person_list_lookup AS (
+    SELECT
+        mpp.project_id,
+        string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ','::text) AS project_team_members
+    FROM moped_proj_personnel mpp
+    JOIN moped_users mu ON mpp.user_id = mu.user_id
+    JOIN moped_proj_personnel_roles mppr ON mpp.project_personnel_id = mppr.project_personnel_id
+    JOIN moped_project_roles mpr ON mppr.project_role_id = mpr.project_role_id
+    WHERE mpp.is_deleted = false AND mppr.is_deleted = false
+    GROUP BY mpp.project_id
+),
+
+funding_sources_lookup AS (
+    SELECT
+        mpf.project_id,
+        string_agg(DISTINCT mfs.funding_source_name, ', '::text ORDER BY mfs.funding_source_name) AS funding_source_name,
+        string_agg(DISTINCT mfp.funding_program_name, ', '::text ORDER BY mfp.funding_program_name) AS funding_program_names,
```

Comment:
@mateoclarke I ran the down migration (`hasura migrate apply --down 1`), and I ran into an error because this new field is in here that prevents that previous version of this view from restoring. Could you remove this one line and then test the down migration? 

I swear that I saw this line go away when we were talking on Friday, but that was a long time ago now. 🙃 Everything else looks great, and I'll approve after testing again. Thanks!

Reply:
this is all good now ✅